src/Nethermind/Nethermind.Network/P2P/Subprotocols/Eth/PooledTxsRequestor.cs

```diff
@@ -69,6 +71,12 @@ public void RequestTransactionsEth68(Action<V66.Messages.GetPooledTransactionsMe
                 int txSize = discoveredTxHashesAndSizes[i].Size;
                 TxType txType = (TxType)discoveredTxHashesAndSizes[i].Type;
 
+                if (txType != TxType.Blob && txSize > _configuredMaxTxSize
+                    || txType == TxType.Blob && txSize > _configuredMaxTxSize * 8 + (long)Eip4844Constants.MaxBlobGasPerBlock)
```

Comment:
```suggestion
                long maxSize = txType != TxType.Blob ? _configuredMaxTxSize : _configuredMaxTxSize * 8 + (long)Eip4844Constants.MaxBlobGasPerBlock;
                if (txSize > maxSize)
```

Also on tx we have `SupportsBlobs` which can be more than just `TxType.Blob` in the future. Would be good to have it as extension method on `TxType`.

Reply:
Maybe also just calculate it once?